SEPACreditTransfer.pas

```diff
@@ -271,6 +273,8 @@ constructor TCreditTransferPaymentInformation.Create;
   fChrgBr           := CHRG_BR_SLEV;
   fDbtrAcct         := TAccountIdentification.Create;
   fDbtrAgt          := TFinancialInstitution.Create;
+
+  fBtchBookg    :=  false; // Set it default to false, we want to have details
```

Comment:
I do not agree on the default. According to the documentation, "true" is the default and for details one needs a special agreement. We cannot assume here that this is what the user has and wants.